lib/js/styles/ItemsList.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+	<div>
+		<div v-for="itemsList in itemsLocal" :key="itemsList.title" class="itemDefinition">
```

Comment:
should this component use the same class name as child components?

Reply:
Cleaned up.